server.js

```diff
@@ -112,12 +138,13 @@ app.prepare().then(() => {
       if (err) {
         throw err;
       }
-
       res.send({
         errorField: {},
         status: 'Message sent',
       });
     });
+
+    sendContactToHubSpot({ firstname, lastname, email, phone });
```

Comment:
@dmytropanontko  you'd rather pass actual value here to keep function interface clean. E.g. like this:
```
sendContactToHubSpot({ firstname: firstName.value, lastname: lastName.value, email: email.value, phone: phone.value.toString() });
```

Reply:
@epaminond okey